MonkeyWrench.Web.UI/EditLane.aspx.cs

```diff
@@ -705,7 +706,16 @@ protected void cmdSave_Click (object sender, EventArgs e)
 		lane.enabled = chkEnabled.Checked;
 		lane.additional_roles = txtRoles.Text;
 		lane.priority = Int32.Parse (lstPriority.SelectedItem.Value);
-		lane.is_protected = chkProtected.Checked;
+		lane.is_protected = chkProtected.Checked;
+
+		// Don't ever pull more than 100 commits
+		try {
+			int commits = Int32.Parse (txtFetchCommits.Text);
+			lane.max_commits_to_fetch = commits >= 100 ? 100 : commits;

```

Comment:
If you want a lane that pulls more than 100 commits, I'm happy to manually write the query that will do so :P I just don't want someone to accidentally put in the wrong number and then have the engineers complain that it takes forever for builds to show up (which they're already pretty vocal about).